packages/core/src/lib/lit-core.ts

```diff
@@ -110,6 +110,7 @@ const BLOCKHASH_SYNC_INTERVAL = 30_000;
 const NETWORKS_REQUIRING_SEV: string[] = [
   LitNetwork.Habanero,
   LitNetwork.Manzano,
+  LitNetwork.DatilTest,
 ];
 
 // The only network we consider entirely static, and thus ignore EPOCH changes for, is Cayenne
```

Comment:
Needs `LitNetwork.DatilTest` here also
Thinking about it, maybe it would make sencse to invert this logic entirel -- do we expect _every_ network going forwards to have epoch changes so that we don't need to update this moving forward.  WDYT?

Reply:
`LitNetwork.DatilTest` is already in the array?

Reply:
I mean in `NETWORKS_WITH_EPOCH_CHANGES`, L117 - L122 -- it has `DatilDev` but not `DatilTest`

Reply:
ahhh! Gotcha! [bb03270](https://github.com/LIT-Protocol/js-sdk/pull/523/commits/bb03270dff09e0ed06e24e0c9b6dcb34d1d04e55)